website/blog/2023-07-23-databend-weekly-103.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "This Week in Databend #103"
+date: 2023-07-23
+slug: 2023-07-23-databend-weekly
+cover_url: 'weekly/weekly-103.png'
+image: 'weekly/weekly-103.png'
+tags: [weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: akoshchiy
+  - name: ariesdevil
+  - name: b41sh
+  - name: ben1009
+  - name: BohuTANG
+  - name: dantengsky
+  - name: everpcpc
+  - name: flaneur2020
+  - name: JackTan25
+  - name: leiysky
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: ZhiHanZ
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Creating Network Policies
+
+**Network Policies** are options for managing network configurations of Databend services. You can use it to restrict access to your account based on user IP address.
```

Comment:
**Network Policies** are used to manage access to Databend services. They can also be utilized to restrict access to a SQL account based on users' IP addresses.